packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -6,21 +6,39 @@ import { AutoCrudContext } from './autocrud-context.js';
 import DeleteButton from './autocrud-delete.js';
 import { AutoCrudDialog } from './autocrud-dialog';
 import css from './autocrud.obj.css';
-import { emptyItem, ExperimentalAutoForm } from './autoform.js';
-import { AutoGrid } from './autogrid.js';
+import { type AutoFormProps, emptyItem, ExperimentalAutoForm } from './autoform.js';
+import { AutoGrid, type AutoGridProps } from './autogrid.js';
 import type { CrudService } from './crud.js';
 import { useMediaQuery } from './media-query';
 import { getProperties } from './property-info.js';
 
 document.adoptedStyleSheets.unshift(css);
 
+export type AutoCrudFormProps<TItem> = Omit<
+  Partial<AutoFormProps<AbstractModel<TItem>>>,
+  'afterSubmit' | 'disabled' | 'item' | 'model' | 'service'
+>;
+
+export type AutoCrudGridProps<TItem> = Omit<
+  Partial<AutoGridProps<TItem>>,
+  'customColumns' | 'model' | 'onActiveItemChanged' | 'refreshTrigger' | 'selectedItems' | 'service'
+>;
```

Comment:
For now I've simply removed the props that are set by auto crud to avoid conflicts with props set by the user, which means they can effectively not be customized. We can figure out later if / how we want to approach this.